### DISCUSSION TITLE
how editor init webworker, there are some problem with my browser

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [microsoft/monaco-editor#4486] [Q&A]
warning in console:
```
PageMain.vue:30  Could not create web worker(s). Falling back to loading web worker code in main thread, which might cause UI freezes. Please see https://github.com/microsoft/monaco-editor#faq
PageMain.vue:30  You must define a function MonacoEnvironment.getWorkerUrl or MonacoEnvironment.getWorker
```

i running website with tauri, also i try to run it in browser (vue project, bundle and serve with vite)
how can i fix it

extra information: i start a discussion in tauri repo, see https://github.com/tauri-apps/tauri/discussions/9595

**Comment by MoYuan-CN** [comment]
platform: Windows 11
browser: Chrome, browser in tauri: Edge(Webview)

npm packages:
- monaco-editor verison: 0.48.0
- tauri-apps/api@1.5.4
- tauri-apps/cli@1.5.12
- vue@3.4.25
- vite@5.2.10

**Reply by NotReeceHarris** [reply]
im assuming you have moved on but this fixed my issue

https://github.com/vitejs/vite/discussions/1791
